Source/DataSources/StaticGroundGeometryPerMaterialBatch.js

```diff
@@ -224,133 +254,117 @@ define([
         if (!primitive.ready) {
             return BoundingSphereState.PENDING;
         }
-
-        var bs = primitive.getBoundingSphere(updater.entity);
-        if (!defined(bs)) {
+        var attributes = primitive.getGeometryInstanceAttributes(updater.entity);
+        if (!defined(attributes) || !defined(attributes.boundingSphere) ||
+            (defined(attributes.show) && attributes.show[0] === 0)) {
             return BoundingSphereState.FAILED;
         }
-
-        bs.clone(result);
+        attributes.boundingSphere.clone(result);
         return BoundingSphereState.DONE;
     };
 
-    Batch.prototype.removeAllPrimitives = function() {
```

Comment:
Why did you change this?  Because it wasn't being used anywhere?  Open a separate PR with this change to all the `Batch` types

Reply:
To be honest I didn't actually notice this, locally `StaticGroundGeometryPerMaterialBatch` was a new file patterned off `StaticGeometryPerMaterialBatch`.

Open here: https://github.com/AnalyticalGraphicsInc/cesium/pull/6554